controllers/modelmesh/const.go

```diff
@@ -45,12 +46,18 @@ const (
 	//The env variable puller uses to configure the config dir (secrets)
 	PullerEnvStorageConfigDir = "STORAGE_CONFIG_DIR"
 
+	//The env variable puller uses to configure the pvc dir (secrets)
+	PullerEnvPVCDir = "PVC_DIR"
+
 	//The puller default port number
 	PullerPortNumber = 8086
 
 	//The puller model mount path
 	PullerModelPath = "/models"
 
+	//The puller model PVC path
+	PullerPVCPath = "/pvc_mounts"
```

Comment:
Could change the name of this field to something like `DefaultPVCMountsDir`? .. i.e. default for the value above.

Just having it be a constant and no env var is fine too, I don't think we'll need to change it.